src/controllers/controller.bar.js

```diff
@@ -114,6 +114,14 @@ function computeFlexCategoryTraits(index, ruler, options) {
 	};
 }
 
+function getMatchingVisibleMetas(scale) {
```

Comment:
Should we create a helper method for this since this method also exists in `scales.linear`?